src/trading/ClaimProceeds.sol

```diff
@@ -0,0 +1,56 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/Controller.sol';
+import 'ROOT/Mutex.sol';
+import 'ROOT/reporting/Interfaces.sol';
+import 'ROOT/extensions/MarketFeeCalculator.sol';
+import 'ROOT/legacy_reputation/SafeMath.sol';
+
+
+/**
+ * @title ClaimProceeds
+ * @dev This allows users to claim their money from a market by exchanging their shares 
+ * AUDIT: Ensure that a malicious market can't subversively cause share tokens to be paid out incorrectly.
```

Comment:
Can you put this as a line comment separate from the doc comment?  When searching for these I usually search for `// [A-Z][A-Z][A-Z][A-Z]` so I can find `// TODO`, `// FIXME` and `// AUDIT` all at once.